src/main/java/ru/yandex/clickhouse/util/ClickHouseRowBinaryStream.java

```diff
@@ -274,6 +274,10 @@ public void writeFloat64Array(double[] doubles) throws IOException {
         }
     }
 
+    public void writeNullable(boolean value) throws IOException {
```

Comment:
At first I expected here to be writeNullable methods for each primitive input type with wrappers instead of primitives. I think this would be the clearest option from client point of view.
However this option is ok as well, but I think this should be called markNextNullable(boolean isNullable), because it doesn't actually write nullable value, but marks that next is nullable.

Reply:
@serebrserg agree, I added javadoc with an example, and renamed method